plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/AbstractSQLDialect.java

```diff
@@ -508,6 +506,22 @@ public String addFiltersToQuery(DBRProgressMonitor monitor, DBPDataSource dataSo
         return SQLSemanticProcessor.addFiltersToQuery(monitor, dataSource, query, filter);
     }
 
+    @Override
+    public void createLike(StringBuilder sql, String mask, boolean negative) {
+        String sqlLike = makeSQLLike(mask);
+        if (sqlLike.contains("%") || sqlLike.contains("_")) {
+            if (negative) sql.append(" NOT");
+            sql.append(" LIKE ?");
+        }  else {
+            sql.append(negative ? "<> ?": "=?");
```

Comment:
I don't like the way these quote marks just appear here without telling the user about it. Let's pass a `String value` parameter to this method (no escaping required).